scripts/zones/Port_Bastok/npcs/Clarion_Star.lua

```diff
@@ -0,0 +1,61 @@
+-----------------------------------
+-- Area: Port Bastok
+-- NPC: Clarion Star
+-- Trust NPC
+-- !pos 81.478 7.500 -24.169 236
+-----------------------------------
+local ID = require("scripts/zones/Port_Bastok/IDs")
+require("scripts/globals/keyitems")
+require("scripts/globals/npc_util")
+require("scripts/globals/quests")
+require("scripts/globals/settings")
+require("scripts/globals/trust")
+-----------------------------------
+
+function onTrade(player,npc,trade)
+    local hasPermit = player:hasKeyItem(tpz.ki.WINDURST_TRUST_PERMIT) or
+                      player:hasKeyItem(tpz.ki.BASTOK_TRUST_PERMIT) or
+                      player:hasKeyItem(tpz.ki.SAN_DORIA_TRUST_PERMIT)
+
+    if hasPermit and trade:getSlotCount() == 1 and tpz.trust.cipherToSpellID[trade:getItemId(0)] ~= nil then
+        player:setLocalVar("TradingTrustCipher", tpz.trust.cipherToSpellID[trade:getItemId(0)])
+        local fameModifier = 0
```

Comment:
There a few different texts, depending on who you trade in. I'll take it out for now because they'll require a lot more mapping to get right.

Reply:
My comment was originally me asking if that was what they were for, but then thought, "That would be incredibly silly, because the trust ID is one of the params - surely those varying lines are tied to that param! _Obviously_ it's for something else, right?"

Yet here we are. Silly SE.